src/api/routes/myGroups.ts

```diff
@@ -13,29 +13,41 @@ import Transcript from "../database/models/Transcript";
 import moment from 'moment';
 import { zGroupCountingTranscripts } from "./groups";
 import { encodeMeetingSubject } from "./meetings";
-import { formatGroupName } from "../../shared/strings";
-import apiEnv from "../apiEnv";
+import { formatGroupName } from "shared/strings";
+import OngoingMeetings from "api/database/models/OngoingMeetings";
+import apiEnv from "api/apiEnv";
 import sleep from "../../shared/sleep";
 import { noPermissionError, notFoundError } from "api/errors";
 
-export function meetingLinkIsExpired(meetingLinkCreatedAt : Date) {
-  // meeting is valid for 31 days after start time
-  // check if the link is created within 30 days
-  return moment() > moment(meetingLinkCreatedAt).add(30, 'days');
+async function generateMeeting(group: Group, availableTmUserId: string) {
+  const now = Math.floor(Date.now() / 1000);
+  const groupName = formatGroupName(group.name, group.groupUsers.length);
+  const res = await createMeeting(availableTmUserId, encodeMeetingSubject(group.id, groupName), now, now + 3600);
+  invariant(res.meeting_info_list.length === 1);
+  return [res.meeting_info_list[0].meeting_id, res.meeting_info_list[0].join_url];
 }
 
+ /**
+   * This api contains procedure @joinMeeting and @list
+   * @joinMeeting will find the group meeting by the input group id
```

Comment:
1. move comments to the top of individual functions.
```
/**
 * ...
 */
joinMeeting: procedure
```

2. don't use `@` to denote symbols. use "`joinMeeting`" instead.